.changeset/weak-foxes-explain.md

```diff
@@ -0,0 +1,86 @@
+---
+'@backstage/create-app': patch
+---
+
+**BREAKING CHANGE**
```

Comment:
The create app creates a one time templated app and gives it to the user.  My reading of this is that this change is not a breaking change to any existing apps, nor does it break how this command works, but just that the resulting TEMPLATE it creates is different than the old ones?  So I don't see this as breaking the `create-app` **itself**, as you don't then use this to go re-recreate an already created app. (instead you backstage-cli versions:bump and read the CHANGELOG, where elsewhere you've already described the required routing changes?)

But in your view, what part did you believe was breaking?

Reply:
Yeah it's just to grab attention. This changelog is mostly a collection of changes that can be done or have to be done to the template when upgrading. Some of the changes are optional and just lister for visibility for those that want to stay up to date. Other changes, like this one, are required for the app to keep functioning after the version bump of the related packages